cc.arduino.IDE2.json

```diff
@@ -0,0 +1,116 @@
+{
+    "app-id": "cc.arduino.IDE2",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "arduino-ide",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=system-bus",
```

Comment:
I'll put this one again since it was on the previous PR.

You need to figure which dbus names this want to access.